database/migrations/2024_09_25_102533_create_integrations_mails_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::create('integrations_mails', static function (Blueprint $table) {
+            $table->uuid('integration_id')->primary();
+            $table->string('type');
```

Comment:
Is this the integration type or the template name?

Reply:
Renamed it to "template_name".